#! /bin/sh /usr/share/dpatch/dpatch-run
## fix-wierd-email.dpatch by Diego IAstrubni <diego.iastrubni@xorcom.com>
##
## DP: stock code from freepbx.org:
## DP: on some places, internal emails are used, and they do not always contains
## DP: a subdomain (a valid email can be "ruth@sales")

@DPATCH@
diff -urNad freepbx-2.1.3.dfsg/amp_conf/htdocs/admin/common/script.js.php /tmp/dpep.7jLVlW/freepbx-2.1.3.dfsg/amp_conf/htdocs/admin/common/script.js.php
--- freepbx-2.1.3.dfsg/amp_conf/htdocs/admin/common/script.js.php	2006-08-22 16:26:29.000000000 +0300
+++ /tmp/dpep.7jLVlW/freepbx-2.1.3.dfsg/amp_conf/htdocs/admin/common/script.js.php	2006-12-06 14:44:49.000000000 +0200
@@ -193,12 +193,20 @@
     if ((i >= sLength) || (s.charAt(i) != "@")) return false;
     else i += 2;
 
+/*
+stock code from freepbx.org:
+on some places, internal emails are used, and they do not always contains
+a subdomain (a valid email can be "ruth@sales")
+
     // look for .
     while ((i < sLength) && (s.charAt(i) != ".")) {
 		i++;
     }
 	if ((i >= sLength - 1) || (s.charAt(i) != ".")) return false;
     else return true;
+*/
+
+    return true;    
 }
 
 // ***************************************************
